receiver/chronyreceiver/README.md

```diff
@@ -0,0 +1,61 @@
+# Chrony Receiver
+
+| Status                   |           |
+| ------------------------ | --------- |
+| Stability                | [alpha]   |
+| Supported pipeline types | metrics   |
+| Distributions            | [contrib] |
+
+The _Chrony Receiver_ is a pure go implementation of the command `chronyc tracking` to allow for
+portability across systems and platforms. All of the data that would typically be captured by
+the tracking command is made available in this receiver, see [documentation](./documentation.md) for
+more details.
+
+## Configuration
+
+### Default
+
+By default, the `chrony` recevier will default to the following configuration:
+
+```yaml
+chrony/defaults:
+  address: unix:///var/run/chrony/chronyd.sock # The default port by chronyd to allow cmd access
+  timeout: 10s # Allowing at least 10s for chronyd to respond before giving up
+
+chrony:
+  # This will result in the same configuration as above
+```
+
+### Customised
+
+The following things can be customised:
+
+- address (required) - the address on where to communicate to `chronyd`
+  - The allowed formats are the following
+    - udp://hostname:port
+    - unix:///path/to/chrony.sock (Please note the triple slash)
+  - chronyd can be configured to allow tracking comamands either by a udp port or via the running chronyd unix socket
+- timeout (optional) - The total amount of time allowed to read and process the data from chronyd
+  - The value of the timeout must be greater than or equal to 1s
```

Comment:
Why such a high minimum?

Reply:
This is came from trial and error on my local talking to a running instance of `chronyd`, unfortunately since I am using a socket instead of a `http.Request`, the timeout is for the entire operation instead of just the request.

This could just be my local being slow but I wanted to make the option sane.

Reply:
I wouldn't then require a timeout to be higher than that, but I'd rather leave a recommendation on the readme. This sounds like an artificial constraint to me.

Reply:
Sounds good